wdl-analysis/src/document/v1.rs

```diff
@@ -1380,28 +1376,47 @@ fn resolve_import(
     }
 
     // Ensure the import has a matching WDL version
-    let import_root = import_node.root().expect("import should have parsed");
     let import_document = import_node
         .document()
         .cloned()
         .expect("import should have been analyzed");
 
-    // Check for compatible imports
-    match import_root.version_statement() {
-        Some(stmt) => {
-            let our_version = stmt.version();
-            if matches!((our_version.text().split('.').next(), importer_version.text().split('.').next()), (Some(our_major), Some(their_major)) if our_major != their_major)
-            {
+    let Some(our_version) = import_document.version() else {
```

Comment:
Shouldn't this be `their_version` instead of `our_version`? `import_document` is the _document being imported_, which I think of as "the other" while "our version" would be the version of the _document doing the importing_, right?

Reply:
Yeah, several names in this function needed work, so I did a pass for consistency in a4e732a